guides/request-validation-php-lumen/example-2/example-2.5.x.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Http\Request;
+use Twilio\Twiml;
+
+$app->post('/voice', ['middleware' => 'TwilioRequestValidator',
+  function () use ($app) {
```

Comment:
Not sure why you're the `use` identifier to pass `$app` into the scope of this anonymous function. I can't see this function using `$app`. Could you provide me some context on this please?